src/mixins/nr-affiliation-mixin.ts

```diff
@@ -134,8 +134,8 @@ export class NrAffiliationMixin extends Mixins(CommonMixin) {
 
   /** Returns business request object. */
   private getBusinessRequest (accountId: number, nr: NameRequestI): BusinessRequest {
-    const name = this.isBenefitCompany(nr) ? 'incorporationApplication' : 'registration'
-    const legalType = this.isBenefitCompany(nr) ? 'BEN' : nr.legalType
+    const name = this.isSupportedEntity(nr) ? 'incorporationApplication' : 'registration'
+    const legalType = this.isSupportedEntity(nr) ? this.entityTypeAlternateCode(nr.entity_type_cd) : nr.legalType
```

Comment:
This'll behave the same as before but now instead of a hard-coded 'BEN', we're checking for FF values.

Reply:
Your new code looks cleaner, for sure 👍